src/components/carousel/carousel.scss

```diff
@@ -0,0 +1,68 @@
+.carousel {
+    width: 100%;
+    height: 100%;
+    position: relative;
+    overflow: hidden;
+
+    &__container {
+        display: flex;
+        height: 100%;
+        width: 300%;
+
+        &__child {
+            width: 100%;
+            
+        }
+
+        &__child img {
+            width: 100%;
+            height: auto;
+        }
+    }
+
+    &__headings {
+        width: 100vw;
+        position: absolute;
+        bottom: 160px;
+        padding-left: 15px;
+    
+        &--title {
```

Comment:
I think it should be __title and __subtitle. -- is for something that could be applied dynamically. If you have component with textSize prop that it would make sense to have --small --medium and --big prefixes, but in this case those are static